README.md

```diff
@@ -9,7 +9,7 @@ This action uploads artifacts (.apk or .ipa) to Visual Studio App Center.
 
 ### `appName`
 
-**Required** App name followed by username e.g. `wzieba/Sample-App`
+**Required** App name preceded by the App Center's username e.g. wzieba/Sample-App
```

Comment:
it wasn't clear for me what the username is. I have an account created using google sign in so in my case there was `some.weirdshit-domaincom`. I thought it might make sense to update the documentation here as well